lib/dsl/lexer.js

```diff
@@ -39,6 +39,11 @@ function createToken(config) {
     }
   }
 
+  // readable labels for diagrams
+  if (_.isString(config.pattern) && config.label === undefined) {
```

Comment:
Checking for type in JS is something I avoid. Here, you check the pattern is a string (so it could be something else).
`config` looks like a custom object.
If it actually is, this is the form this object should have:
```javascript
config = {
  label: '',
  pattern: ...,
  type: ...
}
```
Or create a class and don't deal with this yourself :)

About `config.label`, the recommended check is `!config.label`.

Reply:
I don't understand the alternative you are suggesting, 
* config is a configuration object pattern for the createToken chevrotain API.
* I want to define an easy to read diagrams labels **automatically** for all simple tokens
* Simple tokens are those tokens whose pattern is a simple string instead of a regExp/Function.

The other alternative is to manually write the label for each token but that is more error prone.

I can modify the check to !config.label and check running the eslint --fix on the generated file.